docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+                    "OnError": []
+                },
+                "ActionType": "LoadEntitiesFromSql",
+                "$_isOpen": false,
+                "$_isLoaded": true,
+                "$_isFocus": true,
+                "Definition": {
+                    "IsClientAction": false,
+                    "Settings": {
+                        "Group": "Entities"
+                    }
+                },
+                "Description": "Load Email List"
+            },
+            {
+                "Id": -1,
+                "$_uid": "action15918774716251116",
+                "Parameters": {
+                    "EntityName": "UserDetails",
+                    "Filters": "",
+                    "ContextBehavior": {
+                        "Expression": "",
+                        "Value": "",
+                        "IsExpression": false,
+                        "Parameters": {}
+                    },
+                    "ActionList": [
+                        {
+                            "Id": -1,
+                            "$_uid": "action15918774102278206",
+                            "Parameters": {
+                                "From": "support@plantanapp.com",
+                                "DetermineEmail": "",
+                                "To": "[UserDetails:Email]",
+                                "ToAllUsers": "",
+                                "ReplyTo": "",
+                                "Cc": "",
+                                "Bcc": "",
+                                "Subject": "Created date for [UserDetails:Username]",
+                                "Format": {
+                                    "Expression": "",
+                                    "Value": "Html",
+                                    "IsExpression": false,
+                                    "Parameters": {}
+                                },
+                                "DnnEmailTemplate": {
+                                    "Expression": "",
+                                    "Value": "",
+                                    "IsExpression": false,
+                                    "Parameters": {}
+                                },
+                                "Body": "<p><span style=\"color: rgb(102, 102, 102); font-family: Roboto, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-size: 13px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: 0.1px; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;\">The user: [UserDetails:Firstname] [UserDetails:Lastname] was created on date [UserDetails:CreatedOnDate]</span><!--EndFragment--><br><br><br></p>",

```

Comment:
Can we clean the html a bit so it's more readable?

Reply:
Cleaned